frontend/src/devicelist/ActionMenu.jsx

```diff
@@ -26,6 +28,11 @@ export default class ActionMenu extends React.Component {
     this.setAnchorElement(null);
   }
 
+  handleDelete(deleteId) {
+    DeviceApi.deleteDevice(deleteId);
+    this.handleClose();
+  }
+
```

Comment:
Is this setup for Delete Device via the Device List page?

Reply:
ou yeah it is!

Reply:
ou yeah it is!

Reply:
I see, so should this change be removed and added to that future PR you're mentioning?

Reply:
i can remove it if you and @felixlapierre are willing to re-approve today :eyes:

Reply:
I can re-approve today for sure.